module/VuFindTheme/src/VuFindTheme/View/Helper/SetupThemeResources.php

```diff
@@ -126,16 +126,33 @@ protected function addLinks()
             );
         }
 
-        // If we have a favicon, load it now:
+        // Insert link elements for favicons specified in the `favicons` property of theme.config.php.
+        // If `favicon` is a string then treat it as a single file path to an .ico icon.
+        // If `favicon` is an array then treat each item as an assoc array of html attributes and render a link element for each.
         $favicon = $this->container->getFavicon();
         if (!empty($favicon)) {
             $imageLink = $this->getView()->plugin('imageLink');
-            $headLink(
-                [
-                    'href' => $imageLink($favicon),
-                    'type' => 'image/x-icon', 'rel' => 'shortcut icon'
-                ]
-            );
+            if (is_array($favicon)) {
+                foreach($favicon as $attrs) {
+                    $headLink(
+                        [
+                            'href' => $imageLink($attrs['href']),
+                            'rel' => $attrs['rel'],
```

Comment:
It would probably be a good idea to address the possibility of missing values here -- i.e. either default with null coalescing, skip attributes if no reasonable default can be determined, or skip/throw an exception if a required value like href is missing.

One question is whether we really need to "curate" which attributes pass through. Would it be better to do something like:

```
foreach ($favicon as $attrs) {
    // Convert string into href array
    $attrs = is_array($attrs) ? $attrs : ['href' => $attrs];
    if (!isset($attrs['href'])) {
        continue; // we can't display without an href!
    }
    $attrs['href'] = $imageLink($attrs['href']);
    $attrs['rel'] = $attrs['rel'] ?? 'shortcut icon';
    $attrs['type'] = $attrs['type'] ?? 'image/x-icon';
    $headLink($attrs);
}
```

This is more open-ended if we want to add support for additional attributes via the config file without changing code.

(Note that since this tolerates strings in place of arrays, it could also be refactored to a support method that you could potentially share between both the single-value and multi-value scenarios).